websites/D/Desmos/presence.ts

```diff
@@ -0,0 +1,167 @@
+const desmosPresence = new Presence({
+	clientId: "886254740399349852" //The client ID of the Application created at https://discordapp.com/developers/applications
+  }),
+  /*strings = desmosPresence.getStrings({
+	play: "presence.playback.playing",
+	pause: "presence.playback.paused"
+	//You can use this to get translated strings in their browser language
+  });*/
+  
+var url: string
+var graphing: number = 0
+var pageType: string
+var title: string = ""
+var numEquations: number = 0
+const startTime: number = Date.now()
+  strings = desmosPresence.getStrings({
+    play: "presence.playback.playing",
+    pause: "presence.playback.paused"
+    //You can use this to get translated strings in their browser language
+  });
+
+let url: string,
+ graphing = 0,
+ pageType: string,
+ title = "",
+ numEquations = 0,
+ startTime: number = Date.now();
+
+desmosPresence.on("UpdateData", async () => {
+  /*UpdateData is always firing, and therefore should be used as your refresh cycle, or `tick`. This is called several times a second where possible.
+
+	It is recommended to set up another function outside of this event function which will change variable values and do the heavy lifting if you call data from an API.*/
+	
+  // About the comment above: Is this really recommended? It appears to be not allowed according to the Presence Rules
+  
+  url = document.URL;
+  if (url[url.length-1] === "/") {
+	url = url.substring(0, url.length-1)
+  }
+  var splitUrl = url.split("/")
+  var urlPage = splitUrl[splitUrl.length-1]
+  if (urlPage === "www.desmos.com") { 
+	pageType = "Home Page" 
+  } else { 
+	pageType = urlPage[0].toUpperCase() + urlPage.substring(1) 
+  }
```

Comment:
Fix indenation here, but curlies are not needed as you're just doing one thing
```suggestion
  if (urlPage === "www.desmos.com") pageType = "Home Page" 
  else  pageType = urlPage[0].toUpperCase() + urlPage.substring(1) 
```